meeshkan/serve/mock/storage.py

```diff
@@ -1,28 +1,106 @@
 import logging
+import uuid
+
+from http_types import Request
+from openapi_typed import OpenAPIObject
 
 logger = logging.getLogger(__name__)
 
 
-class StorageManager:
+class Storage:
     def __init__(self):
         self._default = dict()
-        self._storages = dict()
+        self._entities = dict()
 
     @property
     def default(self):
         return self._default
 
-    def get(self, name):
-        return self._storages.get(name)
+    def add_entity(self, name):
+        self._entities[name] = dict()
+
+    def clear(self):
+        self._default.clear()
+        for entity in self._entities.values():
+            entity.clear()
+        logger.debug("Storage cleared")
+
+    def __getattr__(self, x):
+        if x in self.__dict__:
+            return getattr(self, x)
+        return self._entities[x]
+
+    def __getitem__(self, x):
```

Comment:
Aliases like `EntityName = str` and `def __getitem__(self, x: EntityName)` might make this easier to understand and act as a documentation of its own?